tests/fs_test.py

```diff
@@ -11,6 +11,18 @@
 
 from gi.repository import BlockDev, GLib
 
+fs_spec = BlockDev.PluginSpec()
+fs_spec.name = BlockDev.Plugin.FS
+BlockDev.ensure_init(require_plugins=[fs_spec])
```

Comment:
We have intentionally moved library (re)initialization into the test case classes. Please move this to ``setUpClass`` below and skip the tests by adding something like this to the beginning of the tests (methods):
```
if not self.ntfs_avail:
    self.skipTest("skipping NTFS: not available")
```